### ISSUE TITLE
[methodology] Four adoption rulings: active_users scale, reported_traction vocabulary, and two capability records

### ISSUE THREAD

**Issue by ccerv1** [currentai-org/os-ai-map#229]
Four decisions surfaced by the 2026-08-12 adoption audit. None blocks anything; each needs a call rather than more research. Evidence gathered and verified.

## 1. Declare an `active_users` MAU scale

`registry.adoption_bands` is populated from two places only — the four type rubrics (emitted as `usage_volume`) and the stars route. So **`active_users` (23 products) and `reported_traction` (108) have no declared scale anywhere.**

For `active_users` that's a real gap, because **all 23 products have a genuine user figure** and 22 are squeezed into the *download* vocabulary:

| product | actual | records |
|---|---|---|
| `chatgpt` | 900M WAU | `>10M` |
| `poe` | 18M MAU | `>10M` |
| `character-ai` | 20M MAU | `10M-100M` (invented, not a declared band) |

Three products, two labels, no rule. The corpus also needs a band above 100M — Meta AI is at 1B and Gemini at 750M.

**Proposed:** declare it on the adoption route in `signal_routing.yaml` (same place and reasoning as the stars scale — a property of the instrument, not the type), with a `>100M users` top band. Then move `ernie` (200M MAU) onto it.

## 2. Declare `reported_traction` qualitative, and forbid numeric labels on it

Researched across all 108: **only 6 notes contain any user- or customer-style figure at all.** An MAU scale would populate for ~1 in 15. That's not a scale.

But the borrowed download labels are actively wrong where they exist — `perplexity-sonar-api` records `1M-10M` against a note citing **45M+ MAU**; `huggingchat-chat-ui` records `100K-1M` against 1M+ users; `helm` records `100K-1M` against 3,142 downloads.

**Proposed:** mirror `hardware.yaml`'s existing shape —

```yaml
reported_traction:
  qualitative: true
  unit: qualitative reach; no countable artifact and no disclosed user figure
  reach_vocabulary: [early, niche, broad, mass-market]
  bands: []
```

optional, and **make a numeric download-vocabulary label illegal on this instrument.** That prohibition is the part that actually fixes the 68.

**The honest objection:** in hardware the vocabulary is near-collinear with level (`niche` spans 3–4, `broad` 3–4, `mass-market` 4–5), which is the one thing `adoption.md` says `reach` must not be. It earns its place only by carrying "this was a judgment, not a count".

## 3. `personahub` capability 2 — likely wrong

The **only** capability-2 product in a 38-product category where the mode is 3 and nine hold 5. Recorded with `value: null` and `confidence: high` — asserted rather than argued.

Its own cited sources point higher: the paper's Qwen2-7B fine-tune reaching **64.9% on MATH**, and Ai2 conditioning `tulu-3-sft-personas-math` on ~250K PersonaHub personas.

**Decision needed:** re-score, or record why 2 is right.

## 4. `llama` capability cites the wrong release, on a defunct number

`docs/guides/identity.md`: *"adoption, capability — **max** across the tier's releases."* The tier's governing release is Llama 4. The capability value reads **Llama 3.1 405B**.

And the number is unreadable: it records *"AA Intelligence Index 17"*, but Artificial Analysis has rescaled. Our own warehouse (fetched 2026-08-08) reads **Llama 4 Maverick 14.5**, Scout 10.3 — and Llama 3.1 405B isn't listed at all.

Blast radius checked: **only `llama` records a raw AA index number.** The other five AA citations are provider *speed* rankings, a different instrument, unaffected.

Score 3 still holds on current standing. The evidence behind it does not.

---

Full context: PRs #222, #224, #225, #226, #227, #228 and the discussion on #223.

**Comment by ccerv1** [comment]
All four settled and shipped in #231 and #234.

**1. `active_users` scale — declared** (#231). Bands mirror the download thresholds (`>10M` … `<10K`), on Carl's ruling: a level has to mean one magnitude across the whole map, or `adoption` stops being comparable between a package and an app. The alternative considered was one order higher throughout, so ChatGPT at ~900M weekly actives and Poe at ~18M didn't both land at 5; rejected, because a wide top band is the ordinary cost of a five-point scale.

Labels carry `users` even though the thresholds match — *especially* because they match. An unsuffixed `>10M` is ambiguous between two live scales at identical boundaries, and that ambiguity is how a download vocabulary colonised this instrument unnoticed.

**Two levels moved, both up**, and both were the map disagreeing with itself where nothing was looking: `character-ai` read 4 against an invented `10M-100M` while its own ~20M MAU clears the top threshold; `claude-ai` read `1M-10M` over a cited ~19–30M.

**2. `reported_traction` — vocabulary declared, numbers made illegal** (#231). The measurement reversed the original recommendation. The **numeric** labels were the perfectly collinear ones — `100K-1M` was level 3 on all 33 of its records, `1M-10M` level 4 on all 23 — so they carried nothing the level didn't, and carried something false: `aws-neuron` read `100K-1M` beneath a note saying *"no download/user count is published for Neuron."* All 68 stripped; levels stand.

`niche` / `broad` / `mass-market` are declared as a **vocabulary, not a scale** — no level mapping, because those words are the *less* collinear half (85% / 80% / 67%) and a bands table would flatten that. A test asserts at least one word spans more than one level; if that stops being true, they're bands in disguise.

**3. `personahub` capability — re-derived at 4** (#234, in the capability commit). The 2 was never a capability judgment: the note was its **openness note verbatim**, pasted in when openness moved to 2 on 2026-08-01, with the score following it down. A licence says what you may do with a corpus; it says nothing about what training on it produces. Re-derived from the sources already cited — Qwen2-7B on ~1.07M persona math problems reaching 64.9% MATH, and Tulu 3 conditioning on ~250K of its personas. 4 rather than 5 on the category's own calibration.

**4. `llama` capability — re-derived, score unchanged at 3.** It cited Llama 3.1 405B where `identity.md` requires max-across-releases, on an AA index number that no longer exists on that scale. Re-derived on Llama 4 Maverick (14 on v4.1.1, #26/44), with Scout fetched too to confirm the max rather than assume it. The score not moving is the point of re-deriving rather than adjusting.

The copy-paste mechanism behind #3 is now frozen by `tests/test_score_notes.py`, which names the seven products carrying one note across axes so an eighth fails the suite. It doesn't clear that backlog.
